core/src/main/java/com/gentics/mesh/core/data/node/field/list/impl/NodeGraphFieldListImpl.java

```diff
@@ -81,6 +82,9 @@
 			if (node == null) {
 				throw error(BAD_REQUEST, "node_list_item_not_found", item.getUuid());
 			}
+			if (!fieldSchema.isAllowedSchema(node.getSchemaContainer().getName())) {
+				throw error(NOT_FOUND, "node_error_reference_not_allowed_schema", item.getUuid(), fieldKey);
```

Comment:
Add the info from which node the error was caused. This would be very handy when dealing with batch updates or node migration errors.